doc/release/1.10.0-notes.rst

```diff
@@ -74,6 +77,18 @@ Also, the dtype.type of nested structured fields is now inherited.
 New Features
 ============
 
+Reading extra flags from site.cfg
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously customization of compilation of dependency libraries and numpy 
+itself was only accomblishable via code changes in the distutils package.
+Now numpy.distutils reads in the following extra flags from each group of the
+site.cfg:
+  runtime_library_dirs (sets the runtime library directories to override
+    LD_LIBRARY_PATH)
+  extra_compile_args (add extra flags to the compilation of sources)
+  extra_link_args (add extra flags when linking libraries)
```

Comment:
formatting: this should be a reST list, so ``-`` in front of each entry and blank lines above and below the list. and names with underscores should be surrounded by double backticks, otherwise Sphinx interprets those underscores as markup